### PR TITLE
release/celo-phase-1 leftovers

### PR DESCRIPTION
**Pull request by bowd**
### Description

I just realized I had some things on the `release/celo-phase-1` branch that weren't pushed upstream when we merged it.
Essentially: 
- The `mento-core` submodule updated to `tags/v2.0.0`
- The broadcast of the CGP submission.
- Refactoring of the ownership transfer script to do all proxies + implementations - the script was re-ran in this form during the governance approval process because some of the implementations were still owned by our address.
- Adding the correct link to the markdown file on CGP submission.

### Other changes

N/A

### Tested

N/A

### Related issues

N/A

### Backwards compatibility

N/A

### Documentation

N/A

### CODE REVIEW

**Comment by bowd** [comment]
It looks like a lot of commits because it's based on the same branch that was already squash merged on `main`, but the diff is small, those can be ignored as we'll also squash merge this.

File: script/upgrades/MU01/deploy/MU01-03-Governance-Ownership-Transfer.sol [hunk 1]
```diff
@@ -21,32 +25,48 @@ contract MU01_GovernanceOwnershipTransfer is Script {
...
+        address payable proxyAddy = contracts.deployed(proxies[i]);
+        Proxy proxy = Proxy(proxyAddy);
+        if (proxy._getOwner() != governance) {
+          console2.log("%s(%s) ownership transferred to %s", proxies[i], governance);
+          proxy._transferOwnership(governance);
+        }
+      }
+
+      for (uint256 i = 0; i < ownables.length; i++) {
+        address ownableAddy = contracts.deployed(ownables[i]);
```
Comment (ninocomputer) [inline script/upgrades/MU01/deploy/MU01-03-Governance-Ownership-Transfer.sol:62]: ownableAddy :D

File: script/upgrades/MU01/deploy/MU01-03-Governance-Ownership-Transfer.sol [hunk 2]
```diff
@@ -5,12 +5,16 @@ import { Script } from "script/utils/Script.sol";
...
+import { Ownable } from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import { Proxy } from "mento-core/contracts/common/Proxy.sol";
+
+// import { BreakerBoxProxy } from "mento-core/contracts/proxies/BreakerBoxProxy.sol";
+// import { BiPoolManagerProxy } from "mento-core/contracts/proxies/BiPoolManagerProxy.sol";
+// import { BrokerProxy } from "mento-core/contracts/proxies/BrokerProxy.sol";
+// import { MedianDeltaBreaker } from "mento-core/contracts/MedianDeltaBreaker.sol";
+// import { ValueDeltaBreaker } from "mento-core/contracts/ValueDeltaBreaker.sol";
+// import { PartialReserveProxy } from "contracts/PartialReserveProxy.sol";
+
```
Comment (ninocomputer) [inline script/upgrades/MU01/deploy/MU01-03-Governance-Ownership-Transfer.sol:17]:
```suggestion
```

**Review by ninocomputer — APPROVED** [review]
I guess we should remove commented out imports?

**Review by philbow61 — APPROVED** [review]

**Pull request merged**